client/app/domain/redirection/edit/domain-redirection-edit.controller.js

```diff
@@ -138,19 +138,19 @@ angular.module("controllers").controller(
             if (_.isEmpty(this.redirectionTarget)) {
                 this.errors.redirectionTarget = true;
             } else if (this.redirection.isOrt) {
-                this.errors.redirectionTarget = !this.validator.isValidURL(this.redirectionTarget);
+                this.errors.redirectionTarget = !this.validator.url.isValid(this.redirectionTarget);
                 this.errors.targetLength = !this.isTargetValid();
             } else {
                 switch (this.redirection.fieldDisplayType) {
                 case "A":
                     this.errors.redirectionTarget = !this.validator.isValidIpv4(this.redirectionTarget);
```

Comment:
`this.validator.isValidIpv4(this.redirectionTarget)` no longer exists, should use `this.validator.ip.isValid(this.redirectionTarget, { version: 4 });`